cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/DefaultMetricStore.java

```diff
@@ -156,6 +152,26 @@ public void deleteBefore(long timestamp) {
     // todo: implement metric ttl
   }
 
+  private Map<String, String> replaceTagsIfNeeded(Map<String, String> tagValues) {
+    // replace emitted tag names to the ones expected by aggregations
+    Map<String, String> result = Maps.newHashMap();
+    for (Map.Entry<String, String> tagValue : tagValues.entrySet()) {
+      String tagNameReplacement = tagMapping.get(tagValue.getKey());
+      result.put(tagNameReplacement == null ? tagValue.getKey() : tagNameReplacement, tagValue.getValue());
```

Comment:
oh, it's just git for whatever reason shows that it's new method - but there's another was added, not this one:)